src/coreclr/vm/eventtrace.cpp

```diff
@@ -2328,6 +2328,72 @@ enum CallbackProviderIndex
     DotNETRuntimePrivate = 3
 };
 
+// EventFilterType identifies the filter type used by the PEVENT_FILTER_DESCRIPTOR
+enum EventFilterType
```

Comment:
Rather than including the ETW enum I'd focus on values we will accept and what we expect those enumerants to mean. For example:
```suggestion
enum EventFilterType
{
    // data should be pairs of UTF8 null terminated strings all concatenated together.
    // The first element of the pair is the key and the 2nd is the value. We expect one of the
    // keys to be the string "GCSeqNumber" and the value to be a number encoded as text.
    // This is the standard way EventPipe encodes filter values
    StringKeyValueEncoding = 0
    // data should be an 8 byte binary LONGLONG value
    // this is the historic encoding defined by .NET Framework for use with ETW
    LongBinaryClientSequenceNumber = 1
}
```

Reply:
Changed! Thanks